### ISSUE TITLE
Add a heatmap mode to the map

### ISSUE THREAD

**Issue by nodeful** [PotentialWeb/CoronaTab#18]

**Comment by willviles** [comment]
I have reservations over how useful a heatmap visualization actually is for this dataset. Change my mind.

**Comment by nodeful** [comment]
Yeah ur right, besides aesthetics heatmap doesn't provide much value. However, because we have geometry data for a lot of places maybe we can show those instead of circles in some cases. For example Brazil is a huge country but we show it just as a single dot on a map
